src/components/main/Carousel.tsx

```
@@ -4,66 +4,72 @@ import { useState, useEffect } from "react"
+    router.push(`/project/${categoryTopProjects[currentIndex].id}`);
   }
 
-  const currentCard = carouselCards[currentIndex];
+  const currentCard = categoryTopProjects[currentIndex];
+
+  useEffect(() => {
+    loadCategoryTopProjects();
+  }, []);
 
   useEffect(() => {
     const timeout = setTimeout(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: categoryTopProjects.length

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)